README.rst

```diff
@@ -141,6 +141,34 @@ Use storage on any of the many cloud providers `rclone <https://rclone.org/>`_ s
 - implementation of this primarily depends on the specific remote.
 
 
+s3
+~~~
+
+Use storage on an S3-compliant cloud service:
+
+- URL: ``s3:[profile|(access_key_id:access_key_secret)@][schema://hostname[:port]]/bucket/path`` (recommended)
```

Comment:
not sure what "(recommended)" wants to tell me here.

Reply:
Removed that word, had AI do some rephrasing and it decided to add that word.